app/domain/errors.rb

```diff
@@ -449,6 +449,11 @@ module Constraints
         code: "CONJ00069E"
       )
 
+      EmptyAnnotationsListConfigured =  ::Util::TrackableErrorClass.new(
+        msg: "No annotations were given at all. At least one annotation should be configured in the policy",
```

Comment:
You can reuse the same sentence from above:
```suggestion
        msg: "Role must have at least one constraint",
```